core/src/main/java/tc/oc/pgm/tablist/TeamOrder.java

```diff
@@ -33,6 +33,6 @@ public int compare(Team a, Team b) {
     if (team == b) return 1;
 
     // Rest of the teams are ordered by name
-    return a.getName().compareTo(b.getName());
+    return a.getDisplayName().compareTo(b.getDisplayName());
```

Comment:
I have an idea to solve the `getDisplayName` shenanigans. Add `getId(): String` to `Named`